WalletWasabi.Backend/Migrations/20210825090127_ChangeTokenModel.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace WalletWasabi.Backend.Migrations
+{
+    public partial class ChangeTokenModel : Migration
```

Comment:
No need for a migration from the start: delete and db seed files and create the first migration again